steam/steam.go

```diff
@@ -19,7 +21,13 @@ func init() {
 		"name", "link", "img_src", "review", "price_before", "price_now",
 		"discount"}
 	queryAllDiscounts = fmt.Sprintf(
-		"SELECT %s FROM %s", strings.Join(fields, ", "), tableName)
+		"SELECT %s FROM %s", strings.Join(fields, ", "), discountTableName)
+	
+	fields_featured := []string{
+		"name", "link", "img_src", "headline", "price_before", "price_now",
```

Comment:
our [schema for featured game table seems different](https://github.com/this-is-a-bot/digger/blob/master/sql/steam.sql#L14).